book/hardware-battery.md

```diff
@@ -1,38 +1,27 @@
 # Battery Interface
 
-## Overview
-
 Solo's battery is used to power Solo, and needs regular recharging. This document specs out the interface to the Solo battery to build things like chargers and charging pads.
 
 For information on interfacing with the Solo through the Battery Bay, visit the [Battery Bay page](hardware-batterybay.md).
 
 ## Battery Specs
```

Comment:
Is this is the spec of our current battery or is it the "expected minimum spec" of any power supply?

I think probably the former, but either way we should have a sentence making that clear. If this is the spec for our battery, it would be good to explain which values are indicative and which are "necessary". I'm guessing that the only ones we require are the discharge rates, operating voltage and possibly the weight (depending on how that affects aerodynamics).